### ISSUE TITLE
Can't slice binary PLY files

### ISSUE THREAD

**Issue by Harvie** [julienr/meshcut#5]
`examples/ply.py` seems to only work with ASCII PLY files, not with binary PLY.

**Comment by julienr** [comment]
That's right and the parser is pretty minimal. I found `python-plyfile` which claim to support text & binary ply. I haven't tested it though.

I think it's better to use an already existing library if possible than writing a new ply binary parser from scratch.

**Comment by julienr** [comment]
If the number of dependencies is a problem, it seems python-plyfile is a [single file](https://github.com/dranjan/python-plyfile/blob/master/plyfile.py), so you could just copy/paste this in your project
